lighthouse-service/deploy/distributor.yaml

```diff
@@ -30,7 +30,7 @@ spec:
             - name: PUBSUB_URL
               value: 'nats://keptn-nats-cluster'
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.tests.finished'
```

Comment:
Same here. `tests.finished` should be the correct one.

Reply:
see above